rest/resource-server/src/docs/asciidoc/releases.adoc

```diff
@@ -68,6 +68,22 @@ include::{snippets}/should_document_get_releases_with_fields/http-response.adoc[
 ===== Links
 include::{snippets}/should_document_get_releases_with_fields/links.adoc[]
 
+[[resources-release-subscription-list-with-fields]]
+==== Listing releases subscription with fields
+
+A `GET` request will list all of the service's recent releases.
```

Comment:
Change the description as this endpoint provides releases subscription.

Reply:
done.